SlimApp/UI/NewClient/NewClientViewModel.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  NewClientViewModel.swift
+//  SlimApp
+//
+//  Created by Ekaterina Grishina on 11/01/23.
+//
+
+import Foundation
+
+class NewClientViewModel: ObservableObject {
```

Comment:
Maybe here would be better to have a final class in order to restrict future undesired inheritances.